packages/investor-foxy/sample.env

```diff
@@ -1 +1,2 @@
-MNEMONIC='match with local foxy (only used in cli)'
\ No newline at end of file
+MNEMONIC='match with local foxy (only used in cli)'
+ARCHIVE_NODE=https://link-to-archive-node.com
```

Comment:
We don't enforce this via lint (though probably should) - terminate this line with a new line.